src/helpers/queries.js

```diff
@@ -0,0 +1,21 @@
+import { Q } from 'cozy-client'
+import { DOCTYPE_CONTACTS } from './doctypes'
+
+export const queryContactById = {
+  definition: id => () => Q(DOCTYPE_CONTACTS).getById(id),
+  options: id => ({
+    as: `contactById-${id}`
+  })
+}
```

Comment:
why not passing the id directly to `queryContactById` and then using it queryContactById(id).definition ?